MIGRATING.md

```diff
@@ -0,0 +1,17 @@
+## Migrating to Essential Next.js Plugin
+
+This guide is to assist `next-on-netlify` users in their migration to Netlify's [Essential Next.js build plugin](https://github.com/netlify/netlify-plugin-nextjs).
+
+Please visit [this issue](https://github.com/netlify/next-on-netlify/issues/176) to ask questions about migrating and/or the deprecation of `next-on-netlify`.
+
+### Existing NoN users
+
+1. Uninstall `next-on-netlify`
```

Comment:
This may be overkill for our tech-savvy, early adopters and users of this package, but something to consider. Regardless, I suggest adding a period to the end of each item in the numbered list.

```suggestion
1. [Uninstall](https://docs.npmjs.com/uninstalling-packages-and-dependencies) `next-on-netlify`.
```